executor/common_linux.h

```diff
@@ -295,8 +295,11 @@ static void initialize_tun(uint64_t pid)
 	int id = pid;
 
 	tunfd = open("/dev/net/tun", O_RDWR | O_NONBLOCK);
-	if (tunfd == -1)
-		fail("tun: can't open /dev/net/tun");
+	if (tunfd == -1) {
+		info("tun: can't open /dev/net/tun: please enable CONFIG_TUN=y");
```

Comment:
This is fatal in executor and I think we want to remain fatal in executor. Otherwise it will fail and nobody will notice. Fuzzer has already checked that /dev/net/tun is present, so this must not fail.
What about:
```
#ifdef SYZ_EXECUTOR
  fail(...);
#else
  printf(...);
  return;
#endif
```